packages/features/auth/lib/next-auth-options.ts

```diff
@@ -1,5 +1,7 @@
 import type { Membership, Team, UserPermissionRole } from "@prisma/client";
 import { waitUntil } from "@vercel/functions";
+import { OAuth2Client } from "google-auth-library";
+import { calendar_v3 } from "googleapis";
```

Comment:
instead of `import { google } from "googleapis";`

Reply:
I've been working on a similar fix! there's the @googleapis/calendar package (and @googleapis/admin) to also slim down the node_modules. Can confirm this works

Reply:
Potentially you need to use googleapis-common for the OAuth2Client

Reply:
> Potentially you need to use googleapis-common for the OAuth2Client

Oh why is that?

Reply:
> I've been working on a similar fix! there's the @googleapis/calendar package (and @googleapis/admin) to also slim down the node_modules. Can confirm this works

Will handle that in #17811